extensions/wikia/CreatePage/js/CreatePage.js

```diff
@@ -130,15 +135,14 @@ var CreatePage = {
 					}
 				};
 				uiModal.createComponent( createPageModalConfig, function( createPageModal ) {
-					CreatePage.track( { action: 'impression', label: 've-redlink-modal' } );
+					CreatePage.track( { action: Wikia.Tracker.ACTIONS.IMPRESSION, label: 'modal' } );
 
 					createPageModal.bind( 'create', function( event ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "event". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)